src/components/applications/EmptyApplicationsView.js

```diff
@@ -31,6 +31,7 @@ export const EmptyApplicationsView = ({isHR}) => {
                         </Grid>
                         <Grid item>
                             {
+                                !isDev &&
```

Comment:
Może głupie pytanie ale to oznacza że dev nie może zobaczyć widoku z brakiem aplikacji? A jeśli tak to on nie powinien gdzieś wcześniej dostać swala, bo jeśli nie ma aplikacji to nie ma też taskStage'a więc jego hasło będzie niepoprawne, więc backend zwróci jakieś 401?

Reply:
Zobaczy, tylko nie będzie miał tego jednego guzika back to offers